util/libcrypto.num

```diff
@@ -5703,3 +5703,4 @@ OSSL_USER_NOTICE_SYNTAX_it              ?	3_4_0	EXIST::FUNCTION:
 OSSL_INDICATOR_set_callback             ?	3_4_0	EXIST::FUNCTION:
 OSSL_INDICATOR_get_callback             ?	3_4_0	EXIST::FUNCTION:
 OPENSSL_strtoul                         ?	3_4_0	EXIST::FUNCTION:
+X509_STORE_CTX_set0_ocsp_resp           ?	3_4_0	EXIST::FUNCTION:OCSP
```

Comment:
Hint: if you add this line not to the end but little further up,
you won't experience trivial merge conflicts at the time other new functions get added here by other PRs being merged.